README.md

```diff
@@ -298,6 +306,24 @@ console.log("will raise an exception if not in script_hashes.yml!")
 <% end %>
 ```
 
+### Public Key Pins
+
+```
+config.hpkp = {
+  max_age: 60.days.to_i,   # max_age is a required parameter
+  include_subdomains: true, # whether or not to apply pins to subdomains
+  # Per the spec, SHA256 hashes are the only currently supported format.
+  pins: [
+    {sha256: 'b5bb9d8014a0f9b1d61e21e796d78dccdf1352f23cd32812f4850b878ae4944c'},
+    {sha256: '73a2c64f9545172c1195efb6616ca5f7afd1df6f245407cafb90de3998a1c97f'}
+  ],
+  enforce: true,            # defaults to false (report-only mode)
+  report_uri: '//example.com/uri-directive',
```

Comment:
How about something like 

> Be aware that pinning error reporting is governed by the same rules as everything else. If you have a pinning failure that tries to report back to the same host, by definition this will not work. 

@ptoomey3 Do people typically supply a separate host? Do reports go cross-origin? I forget where CSP implementations ended up, but I've had to use a proxy host in the past :sob: to get around this